pkg/capabilities/consensus/ocr3/models.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type config struct {
-	AggregationMethod string      `mapstructure:"aggregation_method" json:"aggregation_method" jsonschema:"enum=data_feeds"`
+	AggregationMethod string      `mapstructure:"aggregation_method" json:"aggregation_method" jsonschema:"enum=data_feeds,enum=reduce"`
```

Comment:
Is this syntax correct? Not "enum=a,b,c"?

Reply:
It's the way that I see it in the example from json schema parser that we use
https://github.com/invopop/jsonschema?tab=readme-ov-file#example